show/main.py

```diff
@@ -119,7 +121,19 @@ def connect_config_db():
     config_db.connect()
     return config_db
 
+def is_gearbox_configured():
+    """
+    Checks whether Gearbox is configured or not
+    """
+    app_db = SonicV2Connector(host='127.0.0.1')
+    app_db.connect(app_db.APPL_DB)
+    keys = app_db.keys(app_db.APPL_DB, '*')
```

Comment:
Would it be better to only check for this specific "GEARBOX_TABBLE..." key instead of getting all the keys in the APPL_DB?  Performance wise it would be more efficient right?

Reply:
Dear @gechiang,

Thank you for review.
I agree with you that using of "GEARBOX_TABLE..." pattern will be more efficient, but in other hand current solution should be more robust. This because in case with "GEARBOX_TABLE..." pattern the issue could be occurred again in case if there will be something wrong with daemons, related to Gearbox, in particular, with GEARSYNCD, which should set config status of Gearbox.
In addition, I have investigated show/main.py and I don't really think that using of "GEARBOX_TABLE..." pattern will significantly increase command execution performance.

Reply:
The performance improvement increases as there are more items in the APP DB.  The current way dump the entire APP DB keys and then search from it.  So the less we have to search from the greater the improvement.

Reply:
Got it! The solution has changed according to your comments. Could you check it?